tayo-blog/src/Pages/Newpost.js

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { APIURL } from '../url';
+export const Newpost = () => {
+  const [details, setDetails] = useState({
+    name: '',
+    title: '',
+    content: '',
+  });
+
+  const set = (name) => {
+    return ({ target: { value } }) => {
+      setDetails((oldValues) => ({ ...oldValues, [name]: value }));
+    };
+  };
```

Comment:
Change this function signature, you can get `name` from `event.target`. And `set` is not a worthy name for this function